packages/sub3suite/PKGBUILD

```diff
@@ -0,0 +1,66 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=sub3suite
+pkgver=v0.0.4.r4.ga6434ce
+pkgrel=1
+pkgdesc='A free, open source, cross platform Intelligence Gathering tool.'
+arch=('any')
+groups=('blackarch' 'blackarch-recon')
+url='https://github.com/3nock/sub3suite'
+license=('GPL3')
+depends=('qt5-base')
+makedepends=('git' 'make' 'qtcreator')
+source=("git+https://github.com/3nock/$pkgname.git"
+        "git+https://github.com/google/gumbo-parser.git"
+        "git+https://github.com/google/breakpad.git")
+sha512sums=('SKIP'
+            'SKIP'
+            'SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+prepare(){
+  cd $pkgname
+
+  git submodule init
```

Comment:
I'm not sure of the way to package git submodules, let's wait @noptrix review

Reply:
> I'm not sure of the way to package git submodules, let's wait @noptrix review

Ok, I tested it. It was the only cleanest and elegant way for submodules. If @noptrix approves, we could create a [PKGBUILD template](https://github.com/BlackArch/blackarch-pkgbuilds) for cases dealing with git submodules.